config/routes.rb

```diff
@@ -379,6 +379,9 @@
           # sometimes underscores get messed up when linked to via SMS
           as: :capture_doc_dashes
 
+      get '/in_person_proofing/ssn' => 'in_person/ssn#show'
+      put '/in_person_proofing/ssn' => 'in_person/ssn#update'
+
```

Comment:
@svalexander  Your next issue is update.  You will need to confirm everything with validation and submission is accurate. I added this in so that I could hit update to make sure UI was accurate. There are slight changes in buttons displayed and text depending on if it is your initial visit to the ssn view.